app/models/solved_challenge.rb

```diff
@@ -11,11 +11,11 @@ class SolvedChallenge < FeedItem
   belongs_to :team, optional: false
 
   def description
-    %(Solved challenge "#{challenge.category.name} #{challenge.point_value}")
+    %(#{super.titleize} "#{challenge.category.name} #{challenge.point_value}")
```

Comment:
Update, it ends up looking like:
```
pry> challenge.description
=> "Solved Challenge \"Web 100\""
```
which is acceptable since it will show in the UI as `Solved Challenge "Web 100"`